gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/hvdc/HvdcLinkProcessor.java

```diff
@@ -241,7 +218,24 @@ private static void copyInformationFromInitialNetwork(Optional<Load> optionalLoa
                 optionalAcLine.get().getTerminal2().connect();
             }
         }
+    }
 
+    private static void copyLoadInformation(final Optional<Load> optionalLoad, final double sideLoadP, final boolean isSideConnected) {
+        if (optionalLoad.isPresent()) {
+            optionalLoad.get().setP0(sideLoadP);
+            if (isSideConnected) {
+                optionalLoad.get().connect();
+            }
+        }
+    }
+
+    private static void copyGeneratorInformation(final Optional<Generator> optionalGen1, final double hvdcInformation, final boolean hvdcInformation1) {
```

Comment:
rename param hvdcInformation by sideTargetP and hvdcInformation1 by isSideConnected. Preferably also rename  optionalGen1 by optionalGen.